tensorflow_probability/python/stats/sample_stats.py

```diff
@@ -469,6 +476,81 @@ def covariance(x,
 
     return cov
 
+CovarianceState = collections.namedtuple('CovarianceState',
+                                         'num_samples, mean, comoment, dtype')
+def init_running_covariance(shape, dtype=tf.float32):
+  """Initialize the Running Covariance State.
+
+  Args:
+    shape: Python `Tuple` representing the shape of incoming samples.
```

Comment:
I believe this code as written also accepts a fully-defined `tf.TensorShape`; check this and augment the docstring to mention that as an option.

Same for `init_running_variance`.

Reply:
Yup! TensorShapes are good